### PR TITLE
Suisin/74077/ts migration of account limits footer portal and tests

### PR DESCRIPTION
**Pull request by suisin-deriv**
## Changes:

Please include a summary of the change and which issue is fixed below:
-   ...

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [X] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/components/src/components/app-settings/app-settings-footer.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+
+const AppSettingsFooter = ({ children }) => (
+    <div data-testid='dt_app_settings_footer' className='dc-app-settings__footer'>
+        {children}
+    </div>
+);
+const AppSettingsFooterLeft = ({ children }) => <div className='dc-app-settings__footer-left'>{children}</div>;
+const AppSettingsFooterRight = ({ children }) => <div className='dc-app-settings__footer-right'>{children}</div>;
```
Comment (amina-deriv) [inline packages/components/src/components/app-settings/app-settings-footer.jsx:9]: why do we have this file in this PR?

File: packages/components/src/components/app-settings/app-settings-footer.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
```
Comment (amina-deriv) [inline packages/components/src/components/app-settings/app-settings-footer.jsx:1]: This file is in jsx

File: packages/account/src/Components/account-limits/account-limits-footer.tsx [hunk 3]
```diff
@@ -2,18 +2,18 @@ import * as React from 'react';
...
 import { Localize } from '@deriv/translations';
-import AccountLimitsContext from './account-limits-context';
+import AccountLimitsContext, { TAccountLimitsContext } from './account-limits-context';
 
 const AccountLimitsFooterPortal = () => {
-    const { footer_ref, toggleOverlay } = React.useContext(AccountLimitsContext);
+    const { footer_ref, toggleOverlay } = React.useContext<TAccountLimitsContext>(AccountLimitsContext);
 
     return createPortal(
         <a className='link link--prominent' onClick={toggleOverlay} data-testid='footer_text'>
```
Comment (amina-deriv) [inline packages/account/src/Components/account-limits/account-limits-footer.tsx:11]: can u change the testid dt_footer_text

Comment (suisin-deriv) [reply packages/account/src/Components/account-limits/account-limits-footer.tsx:11]: sure 👌

File: packages/components/src/components/app-settings/app-settings-footer.jsx (hunk 1 above)
Comment (suisin-deriv) [reply packages/components/src/components/app-settings/app-settings-footer.jsx]: will update the PR to latest pull request 🙏

**Review by amina-deriv — APPROVED** [review]

**Review by jim-deriv — APPROVED** [review]

**Pull request merged**